src/core/model.js

```diff
@@ -241,6 +246,9 @@ export default class Model extends Object3D {
    */
   /* eslint-disable max-statements */
   render(uniforms = {}) {
+    if (window.__SEER_INITIALIZED__) {
+      addModel(this);
```

Comment:
That was because in the constructor, it appears sometimes that id's are not yet properly initialized, or using temporary names like `Model-1`, but maybe I'm doing something wrong?